spec/features/categories/show_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+describe "Browsing Items by category" do
+  scenario "see all items assigned to that category" do
+    category_1, category_2 = create_list(:category, 2)
```

Comment:
I'm also leaving this. I will make an issue for updating the style guide so this stops happening. Not sure how to do that yet.